.github/workflows/release.yaml

```diff
@@ -32,3 +32,41 @@ jobs:
       - name: test-image
         run: |
           VERSION=3.6.99 ./scripts/test_images.sh
+      - name: save-image
+        run: |
+          docker image save -o /tmp/etcd-img.tar gcr.io/etcd-development/etcd
+      - name: upload-image
+        uses: actions/upload-artifact@v3
+        with:
+          name: etcd-img
+          path: /tmp/etcd-img.tar
+          retention-days: 1
+  trivy-scan:
+    needs: main
+    strategy:
+      fail-fast: false
+      matrix:
+        platforms: [amd64, arm64, ppc64le, s390x]
```

Comment:
Do we need multiple platforms here?  It looks like the same image would be loaded each time regardless?  I might be missing something though.

Reply:
It's multi-arch image.  FYI. https://github.com/etcd-io/etcd/blob/547aa643e69d55328b203f40ccec50256b44158e/scripts/release.sh#L270-L276


Regarding whether to verify all platforms, it seems not necessary; I am not sure whether it's possible that some CVEs only exist for some platform/archs. But it won't do any harm to verify all archs.